src/panels/lovelace/components/hui-image.ts

```diff
@@ -173,7 +175,8 @@ export class HuiImage extends LitElement {
     if (this.cameraImage && this._attached) {
       this._cameraUpdater = window.setInterval(
         () => this._updateCameraImageSrc(),
-        UPDATE_INTERVAL
+        Number(this.refreshRate ? this.refreshRate : DEFAULT_REFRESH_RATE) *
+          1000
       );
```

Comment:
Don't do the `Number` conversion, the values are numbers

Reply:
I originally coded it with "number", however the value was never saved.  Maybe I missed somewhere I needed to change.